src/content/configuration/other-options.mdx

```diff
@@ -551,6 +551,63 @@ Options for infrastructure level logging.
 
 `object = {}`
 
+### appendOnly
+
+<Badge text='5.31.0+' />
+
+`boolean`
+
+Only appends lines to the output. Avoids updating existing output e. g. for status messages. This option is only used when no custom [`console`](#console) is provided.
+
+**webpack.config.js**
+
+```javascript
+module.exports = {
+  //...
+  infrastructureLogging: {
+    appendOnly: true,
+  },
+};
+```
+
+### colors
+
+<Badge text='5.31.0+' />
+
+`boolean`
+
+Enable/Disable colorful output. This option is only used when no custom [`console`](#console) is provided.
+
+**webpack.config.js**
+
+```javascript
+module.exports = {
+  //...
+  infrastructureLogging: {
+    colors: true,
+  },
```

Comment:
```suggestion
infrastructureLogging: {
	colors: true,
	level: "verbose"
},
plugins: [
	compiler => {
		const logger = compiler.getInfrastructureLogger("MyPlugin");
		logger.log("this output will be colorful");
	}
],
```
Would be helpful to add a full example, otherwise it's sort of useless.